index.html

```diff
@@ -418,5 +418,116 @@ <h2>Transferable MediaStreamTrack</h2>
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
   </section>
+  <section>
+    <h2>Lighting correction</h2>
+    <section>
```

Comment:
Somewhere in here it should explain what lighting correction is supposed to do. If it's a boolean, it would be nice if at least all implementations are correcting in the same direction, even though their results may be different.

Reply:
I documented the direction.